lib/perl/Genome/VariantReporting/Command/Wrappers/IgvSession.pm

```diff
@@ -72,8 +73,10 @@ sub bams {
 
 sub bam_paths {
     my $self = shift;
-    #return join(',', map {File::Spec->join($ENV{GENOME_SYS_SERVICES_FILES_URL}, $_)} values %{$self->bams});
-    return join(',', values %{$self->bams});
+    return join(',', map {
+        my $u = URI->new($_, $ENV{GENOME_SYS_SERVICES_FILES_URL});
```

Comment:
`$ENV{GENOME_SYS_SERVICES_FILES_URL}` doesn't need to show up twice.  Instead it can be either:

```
my $u = URI->new_abs($_, $ENV{GENOME_SYS_SERVICES_FILES_URL});
$u->as_string;
```

or:

```
my $u = URI->new($_);
$u->abs($ENV{GENOME_SYS_SERVICES_FILES_URL})->as_string;
```